policy/authorization/webhooks.rego

```diff
@@ -0,0 +1,16 @@
+package authorization
+
+deny[{
+	"id": "webhookconfigurations-dhc-prefix",
+	"resolution": "You're not allowed to create/update/delete ValidatingWebhookConfigurations & MutatingWebhookConfigurations starting with 'system'",
+}] {
+	input.kind = "SubjectAccessReview"
```

Comment:
Why not make `SubjectAccessReview` data model follow the same model of 'admit` policies instead of using `input` ?

Reply:
I guess that;s possible. The only problem is that not every SubjectAccessReview directly matches to the existing pattern (NonResourceAttributes). But as discussed a Mapping is possible and I'll update the PR.